src/sqlancer/SQLProviderAdapter.java

```diff
@@ -20,25 +21,22 @@ public LoggableFactory getLoggableFactory() {
     }
 
     @Override
-    protected void checkViewsAreValid(G globalState) {
+    protected void checkViewsAreValid(G globalState) throws SQLException {
         List<? extends AbstractTable<?, ?, ?>> views = globalState.getSchema().getViews();
         for (AbstractTable<?, ?, ?> view : views) {
             SQLQueryAdapter q = new SQLQueryAdapter("SELECT 1 FROM " + view.getName() + " LIMIT 1");
             try {
                 if (!q.execute(globalState)) {
-                    dropView(globalState, view.getName());
+                    throw new AssertionError();
```

Comment:
Why do we throw an `AssertionError` here rather than trying to drop the view? The `SELECT` might fail when the view is invalid.